tools/roslaunch/src/roslaunch/cached.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
Please rename the file since `cached` does not clarify what the file is providing. Maybe `resource_cache`?